.github/workflows/playwright.yml

```diff
@@ -1,19 +1,198 @@
 name: Playwright Tests
 on:
-  workflow_run:
-    workflows: ["ci"]
-    types:
-      - completed
+  push:
+    branches:
+      - master
+    tags:
+      - "v*"
+  pull_request:
   workflow_dispatch:
 
 env:
   TERM: xterm
   GO_VERSION: 1.19.6
-  NODE_VERSION: 16.20.2
 
 jobs:
   playwright-run:
     runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11.13
+        env:
+          POSTGRES_USER: mmuser
+          POSTGRES_PASSWORD: mostest
+          POSTGRES_DB: mattermost_test
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      minio:
+        image: minio/minio:RELEASE.2019-10-11T00-38-09Z
+        env:
+          MINIO_ACCESS_KEY: minioaccesskey
+          MINIO_SECRET_KEY: miniosecretkey
+          MINIO_SSE_MASTER_KEY: "my-minio-key:6368616e676520746869732070617373776f726420746f206120736563726574"
+      inbucket:
+        image: mattermost/inbucket:release-1.2.0
+        ports:
+          - 10080:10080
+          - 10110:10110
+          - 10025:10025
+      elasticsearch:
```

Comment:
Do minio and elasticsearch to be used during testing? If not, maybe these can be removed?